### ISSUE TITLE
Why train accuracy is smaller than test accuracy ?

### ISSUE THREAD

**Issue by hagianga21** [loicland/superpoint_graph#121]
Hi Loic,
Why train accuracy is smaller than test accuracy? I am so surprised when I run your validate code as `CUDA_VISIBLE_DEVICES=1 python learning/main.py --dataset sema3d --SEMA3D_PATH 'datasets/semantic3d' --epochs 450 --lr_steps '[350, 400]' --test_nth_epoch 100 \
--model_config 'gru_10,f_8' --ptn_nfeat_stn 11 --nworkers 2 --odir "results/3dseman/standard" --batch_size 1`

**Comment by loicland** [comment]
It's not that weird, SPG has very few parameters, and the validation set is small.

You can always try to add more epochs but I an not sure it will increase the performance on the server-side test set.

**Comment by hagianga21** [comment]
Well, thank you, it seems magical to me because I 've never seen this before.
